### PR TITLE
Service tests - getUserId

### PR DESCRIPTION
**Pull request by KaitlynSalemi**

### CODE REVIEW

File: src/test/java/com/cag/cagbackendapi/services/user/UserServiceTest.kt [hunk 1]
```diff
@@ -77,4 +78,51 @@ class UserServiceTest {
         verify(userDao).saveUser(inputUser)
         verifyNoMoreInteractions(userDao)
     }
+
+    @Test
+    fun getByUserId_validUser_success() {
+        val userId = "123e4567-e89b-12d3-a456-426614174000"
```
Comment (prebollar15) [inline src/test/java/com/cag/cagbackendapi/services/user/UserServiceTest.kt:84]: lines 84+85 can be replaced with 'val userId = UUID.randomUUID()'

Comment (prebollar15) [inline src/test/java/com/cag/cagbackendapi/services/user/UserServiceTest.kt]: Lines 84+85 can be replaced with `val userId = UUID.randomUUID()`

**Review by prebollar15 — CHANGES_REQUESTED** [review]
Within userServiceTest.kt, could you add tests for a null/empty userid getting passed though and if the userDao returns a null value? That will cover the 3 exception scenarios that are present within the getUserById userService

File: src/test/java/com/cag/cagbackendapi/services/user/UserServiceTest.kt [hunk 2]
```diff
@@ -77,4 +78,51 @@ class UserServiceTest {
...
+        val badRequestException = BadRequestException(DetailedErrorMessages.INVALID_USER_ID, null)
+
+        val actualException = assertThrows<BadRequestException> {
+            userService.getByUserId(userId)
+        }
+
+        assertEquals(badRequestException.message, actualException.message)
+
+        verifyZeroInteractions(userDao)
+    }
```
Comment (CodyNicholson) [inline src/test/java/com/cag/cagbackendapi/services/user/UserServiceTest.kt:110]: We need one more test for UserNotFound

**Review by CodyNicholson — APPROVED** [review]

**Review by prebollar15 — APPROVED** [review]
